osbs/api.py

```diff
@@ -502,6 +525,31 @@ def create_build(self, **kwargs):
         return self.create_prod_build(**kwargs)
 
     @osbsapi
+    def create_worker_build(self, *args, **kwargs):
+        """
+        Create a worker build
+
+        Pass through method to create_prod_build with the following
+        modifications:
+            - platform param is required
+            - release param is required
+            - inner template set to worker_inner.json if not set
+            - outer template set to worker.json if not set
+            - customize configuration set to worker_customize.json if not set
+
+        :return: BuildResponse instance
+        """
+        for required in ('platform', 'release'):
+            if not kwargs.get(required):
+                raise ValueError('Worker build requires %s param' % required)
```

Comment:
this will raise the 1st error only, it would be better to report all problems and raise after